ci-operator/step-registry/openshift/upgrade/rollback/aws/loki/openshift-upgrade-rollback-aws-loki-workflow.yaml

```diff
@@ -0,0 +1,25 @@
+workflow:
+  as: openshift-upgrade-rollback-aws-loki
+  steps:
+    pre:
+      - ref: ipi-install-monitoringpvc
+      - ref: ipi-install-loki
+      - chain: ipi-aws-pre-stableinitial
+    test:
+      - ref: openshift-e2e-test
+    post:
+      - ref: openshift-e2e-test-capabilities-check
+      - ref: gather-loki
+      - chain: ipi-aws-post
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: "release:initial"
+      OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: "release:latest"
+    env:
+      TEST_COMMAND: "run-upgrade"
+      TEST_SUITE: "all"
+      DELETE_MC: "false"
+      TEST_OPTIONS: "abort-at=99"
+  documentation: |-
+    The Openshift E2E AWS workflow executes the upgrade end-to-end test suite on AWS with a default cluster
+    configuration and loki as a log collector, exercising an upgrade during the process and aborting it at
+    99% so that it will rollback to initial release.
```

Comment:
`abort-at` here means "after 99 mins of waiting start an upgrade back to original release". Please update the docs accordingly.

Reply:
I wasn't exactly sure what 99 quantified, but I did find this:
https://github.com/openshift/openshift-tests/blob/e0d37add24c5dfc34497ad334abc6f02be96312a/cmd/extended-platform-tests/openshift-tests.go#L158

is it 99m or 99%. still learning here.